includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -997,11 +962,9 @@ public function ajax_update_shipping_method() {
 
 		WC()->cart->calculate_totals();
 
-		$product_view_options      = filter_input_array( INPUT_POST, [ 'is_product_page' => FILTER_SANITIZE_STRING ] );
-		$should_show_itemized_view = ! isset( $product_view_options['is_product_page'] ) ? true : filter_var( $product_view_options['is_product_page'], FILTER_VALIDATE_BOOLEAN );
+		$itemized_display_items = filter_input( INPUT_POST, 'is_product_page', FILTER_VALIDATE_BOOLEAN );
 
-		$data           = [];
-		$data          += $this->build_display_items( $should_show_itemized_view );
+		$data           = $this->build_response( $itemized_display_items, true ); // `ajax_update_shipping_method` is called only when the user has added a shipping address
```

Comment:
Another case of the above comment, just to keep things easier for others:

> `ajax_update_shipping_method` is called only when the user has added a shipping address

I can understand this because I know about the context, but it's not clear we're talking about the `true` param being passed.

Just complementing my previous comment: I believe there's no need to explain the **why** in here yet, but it's definitely a good idea to explain it in `build_response`. Perhaps a longer blurb explaining when it should be `true` and when it should be `false`. 👍 

Not that this is a rule or anything. There are many exceptions I suppose. But just based on how current code/comments seem to be structured right now.

Reply:
done 0f1605f